accounts/CHANGELOG.md

```diff
@@ -0,0 +1,444 @@
+
+   https://github.com/owncloud/product/issues/222
+   https://github.com/owncloud/ocis/accounts/pull/116
+
+# Changelog for [0.4.1] (2020-08-27)
+
+The following sections list the changes in ocis-accounts 0.4.1.
+
+[0.4.1]: https://github.com/owncloud/ocis/accounts/compare/v0.3.0...v0.4.1
+
+## Summary
+
+* Bugfix - Adapting to new settings API for fetching roles: [#96](https://github.com/owncloud/ocis/accounts/pull/96)
+* Change - Create account api-call implicitly adds "default-user" role: [#173](https://github.com/owncloud/product/issues/173)
+
+## Details
+
+* Bugfix - Adapting to new settings API for fetching roles: [#96](https://github.com/owncloud/ocis/accounts/pull/96)
+
+   We fixed the usage of the ocis-settings endpoint for fetching roles.
+
+   https://github.com/owncloud/ocis/accounts/pull/96
+
+
+* Change - Create account api-call implicitly adds "default-user" role: [#173](https://github.com/owncloud/product/issues/173)
+
+   When calling CreateAccount default-user-role is implicitly added.
+
+   https://github.com/owncloud/product/issues/173
+
+# Changelog for [0.3.0] (2020-08-20)
+
+The following sections list the changes in ocis-accounts 0.3.0.
+
+[0.3.0]: https://github.com/owncloud/ocis/accounts/compare/v0.4.0...v0.3.0
+
+## Summary
+
+* Bugfix - Atomic Requests: [#82](https://github.com/owncloud/ocis/accounts/pull/82)
+* Bugfix - Unescape value for prefix query: [#76](https://github.com/owncloud/ocis/accounts/pull/76)
+* Change - Adapt to new ocis-settings data model: [#87](https://github.com/owncloud/ocis/accounts/pull/87)
+* Change - Add permissions for language to default roles: [#88](https://github.com/owncloud/ocis/accounts/pull/88)
+
+## Details
+
+* Bugfix - Atomic Requests: [#82](https://github.com/owncloud/ocis/accounts/pull/82)
+
+   Operations on the file system level are now atomic. This happens only on the provisioning API.
+
+   https://github.com/owncloud/ocis/accounts/pull/82
+
+
+* Bugfix - Unescape value for prefix query: [#76](https://github.com/owncloud/ocis/accounts/pull/76)
+
+   Prefix queries also need to unescape token values like `'some ''ol string'` to `some 'ol
+   string` before using it in a prefix query
+
+   https://github.com/owncloud/ocis/accounts/pull/76
+
+
+* Change - Adapt to new ocis-settings data model: [#87](https://github.com/owncloud/ocis/accounts/pull/87)
+
+   Ocis-settings introduced UUIDs and less verbose endpoint and message type names. This PR
+   adjusts ocis-accounts accordingly.
+
+   https://github.com/owncloud/ocis/accounts/pull/87
+   https://github.com/owncloud/ocis/settings/pull/46
+
+
+* Change - Add permissions for language to default roles: [#88](https://github.com/owncloud/ocis/accounts/pull/88)
+
+   Ocis-settings has default roles and exposes the respective bundle uuids. We now added
+   permissions for reading/writing the preferred language to the default roles.
+
+   https://github.com/owncloud/ocis/accounts/pull/88
+
+# Changelog for [0.4.0] (2020-08-20)
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)